### ISSUE TITLE
Tracking in a course : Sorting of the list of the Students (bug)

### ISSUE THREAD

**Issue by tanchon** [chamilo/chamilo-lms#2301]
Not the same issue as #2299 

### Current behavior / Resultado actual / Résultat actuel

In a course, when i display the list of students in the "tracking tool", i'm trying to sort the list. 

To do that, i click on a column heading (firstname or lastname). But if i click on the firstname, the list is sorted by the lastname. And when i click on the lastname, the list is sorted by the firstname.

### Expected behavior / Resultado esperado / Résultat attendu

If i click on the firstname, the list is sorted by the firstname. And when i click on the lastname, the list is sorted by the lastname.

### Steps to reproduce / Pasos para reproducir / Étapes pour reproduire

In a course, in the tracking tool.

### Chamilo Version / Versión de Chamilo / Version de Chamilo
1.11.4

**Comment by tanchon** [comment]
This issue disappeared when i apply the advise of jmontoyaa concerning another "sorting issue" (see #2299). 
I activate the "name_order_conventions" in configuration.php and applied "last_name" ('french' => array('format' => 'title last_name first_name', 'sort_by' => 'last_name')).

Everything is ok now  :  if i click on the firstname, the list is sorted by the firstname. And when i click on the lastname, the list is sorted by the lastname.

**But i don't activate this option, the default configuration seems to have a negative impact, as mentioned in my first post above.** : in this case, the issue remains.

**Comment by AngelFQC** [comment]
@tanchon please try to reproduce in 11.chamilo.org 
It works for me in https://11.chamilo.org/main/tracking/courseLog.php?cidReq=123333&id_session=0&gidReq=0&gradebook=0&origin=

**Comment by tanchon** [comment]
@AngelFQC 

I tried in in 11.chamilo.org. It's ok... except that column headings seems to have been "switched" (in the lastname column, there are firstnames and in the firstname column, there are lastnames).

I tried your changes on the tracking.lib.php file, in my 1..11.4 and 1.11.6a plaforms, and everything is ok : the issue seems to be solved and i don't have the "switched" columns issue in the 11.chamilo.org (it's strange ?!).

Thanks a lot !
